configs/resident/tour_mode_choice.csv

```diff
@@ -305,7 +305,7 @@ util_TNC_PRM_TNC_wait_time,TNC_PRM - TNC wait time,"@np.where(df.nev_prm_access_
 util_TNC_PRM_TNC_fare,TNC_PRM - TNC fare,"@(~df.nev_prm_access_available) * (~df.microtransit_prm_access_available) * ((np.maximum(TNC_single_baseFare*2 + ((odt_skims['KNROUT_PRM_ACC'] + dot_skims['KNRIN_PRM_EGR'])/60 * driveSpeed) * TNC_single_costPerMile + (odt_skims['KNROUT_PRM_ACC'] + dot_skims['KNRIN_PRM_EGR']) * TNC_single_costPerMinute, TNC_single_costMinimum*2)*100)/df.cost_sensitivity)",,,,,,,,,,,,,,,,coef_income,,,,,,,
 util_TNC_PRM_Walk_egress_time_(at_attraction_end),TNC_PRM - Walk egress time (at attraction end),"@np.where(df.nev_prm_egress_available, df.nev_prm_egress_time, np.where(df.microtransit_prm_egress_available, df.microtransit_prm_egress_time, 2*df.dest_prm_time)) * df.time_factor",,,,,,,,,,,,,,,,coef_acctime,,,,,,,
 util_TNC_PRM_wait_egress_time_(at_attraction_end),TNC_PRM - Egress mt/nev wait time (at attraction end),"@np.where(df.nev_prm_egress_available, 2*nevWaitTime, np.where(df.microtransit_prm_egress_available, 2*microtransitWaitTime, 0)) * df.time_factor",,,,,,,,,,,,,,,,coef_wait,,,,,,,
-util_TNC_PRM_Walk_other_time,TNC_PRM - Walk other time,@ (odt_skims['KNROUT_PRM_XFERWALK'] + dot_skims['KNRIN_PRM_XFERWALK']) *df.time_factor,,,,,,,,,,,,,,,,coef_xwalk,,,,,,,
+util_TNC_PRM_Walk_other_time,TNC_PRM - Walk other time,@(odt_skims['KNROUT_PRM_XFERWALK'] + dot_skims['KNRIN_PRM_XFERWALK']) *df.time_factor,,,,,,,,,,,,,,,,coef_xwalk,,,,,,,
```

Comment:
@i-am-sijia I did not quite understand this small fix for sharrow. Was it just deleting a space?